### PR TITLE
Installing rspec, shoulda matcher

### PR DESCRIPTION
**Pull request by MiriamSal**
## PT Link
https://www.pivotaltracker.com/n/projects/#160919683

## User story
```
As a DevTeam 
To be able to do acceptance tests
We need to install rspec
```

## Task
- Install rspec
- Install shoulda matchers
- Set generators to false
- Also removed comments in rails_helper.rb

## Gems
- gem 'factory_bot_rails'
 - gem 'pry-byebug'
 - gem 'pry-rails'
 - gem 'rspec-rails'
 - gem 'shoulda-matchers'

### CODE REVIEW

**Review by RhonGabriel — COMMENTED** [review]
remove comments in spec helper, thanks

**Comment by MiriamSal** [comment]
Cleaned up spec helper

**Comment by MiriamSal** [comment]
Removed gem 'jbuilder', '~> 2.5'

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  config.filter_rails_from_backtrace!
+
+  Shoulda::Matchers.configure do |config|
+    config.integrate do |with|
+      with.test_framework :rspec
+      with.library :rails 
+  end
+
+end
```
Comment (faraznaeem) [inline spec/rails_helper.rb:27]: please format your code

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  end
+
+  
+  config.profile_examples = 10
+
+  
+  config.order = :random
+
+
+  Kernel.srand config.seed
```
Comment (faraznaeem) [inline spec/spec_helper.rb:27]: format code

**Review by faraznaeem — CHANGES_REQUESTED** [review]

File: spec/spec_helper.rb (hunk 2 above)
Comment (MiriamSal) [reply spec/spec_helper.rb:27]: Done

Comment (MiriamSal) [reply spec/spec_helper.rb]: Done

File: spec/rails_helper.rb (hunk 1 above)
Comment (MiriamSal) [reply spec/rails_helper.rb:27]: Done

**Pull request merged**